DoublyLinkedList/doubly_linked_list.h

```diff
@@ -0,0 +1,193 @@
+#include <iostream>
+#include <stdio.h>
+using namespace std;
+// Node Creation
+class doubly_linked_list
+{ //class
+        int data;
+        doubly_linked_list *next;
+        doubly_linked_list *prev;
+        doubly_linked_list *head=NULL;
+
+
+public:                                                      //Access specifier
+        void push(int value); // Insertion of value at front
+
+        void append(int value); //Insertion of value at end
+
+        void Insert(int value, int location); //for inserting at location
```

Comment:
consistently name methods.. some are starting in lowercase and some are in capital and some are camelcase. 

use snake case with lower case letters.

Reply:
ok sir